src/Abstractions/TaskOptions.cs

```diff
@@ -109,3 +125,29 @@ public record StartOrchestrationOptions(string? InstanceId = null, DateTimeOffse
     /// </summary>
     public IReadOnlyDictionary<string, string> Tags { get; init; } = ImmutableDictionary.Create<string, string>();
 }
+
+/// <summary>
+/// Options for calling activities from an orchestrator.
+/// </summary>
+public record CallActivityOptions : TaskOptions

```

Comment:
Why do we need this class if the base class already supports Tags? Shouldn't the `CallActivityAsync` implementation just reference `TaskOptions.Tags` instead of `CallActivityOptions.Tags`?

Reply:
mainly keeping this for semantic separation and clarity. context.CallActivityAsync(..., CallActivityOptions options) is self documenting and consistent with suborchoptions (even now callactivityoptions dont have unique fields in itself)